lib/storage/sqlite.php

```diff
@@ -54,21 +55,16 @@ public function __construct( string $db ) {
 SQL;
 
 			verbose( "SQLite: $sql" );
-			$create = self::$db->exec( $sql );
+			self::$db->exec( $sql );
 		}
 	}
 
 	private function _remove_key( string $key ): bool {
 		$query = self::$db->prepare( 'DELETE FROM storage WHERE "key" = :key' );
 		$query->bindValue( ':key', $key, SQLITE3_TEXT );
 		verbose( "SQLite: {$query->getSQL( true )}" );
-		$result = $query->execute();
 
-		if ( $result === false ) {
-			return false;
-		} else {
-			return true;
-		}
+		return (bool) $query->execute();
```

Comment:
I can appreciate removing the if/else combination.  However, return should be a simple variable/value instead of an expression.  This makes it easier to work with the results of `$query->execute()` later on, if needed ( like debugging an unexpected behavior ).